calicoctl/commands/node_darwin.go

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2016 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package commands
+
+import (
+	"fmt"
+	"os"
+)
+
+// Node function is a switch to node related sub-commands
+func Node(args []string) {
+	fmt.Println("node sub-command is not available on darwin")
```

Comment:
Could we make this:   Error executing command: calicoctl node commands are not available on this OS.